### ISSUE TITLE
Correct heading order for the Sources page

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#1617]
## Problem

The heading order on the Sources page (https://search.openverse.engineering/sources) is currently incorrect. `<h2>` is skipped.

## Description

The heading order should be logical (h1-h6), but the visual styles should still match the designs.

**Comment by ramadanomar** [comment]
I'd love to fix this!
